helm/cluster-api/templates/core/certificate.yaml

```diff
@@ -2,19 +2,22 @@ apiVersion: cert-manager.io/v1
 kind: Certificate
```

Comment:
Currently in AWS we are using a single certificate for all CAPI providers/components (core, bootstrap, controlplane). Here we align with upstream and use a certificate per provider/component.